Command/RunCommand.php

```diff
@@ -428,9 +429,25 @@ private function getCommandProcessBuilder()
             $pb->add('exec');
         }
 
+        // Localize the `console` command
```

Comment:
```php
realpath($_SERVER['SCRIPT_FILENAME'])
// or
realpath($_SERVER['argv'][0])
```
should do the trick, whether the `php` executable is mentioned or not.

This looks safer than expecting to find a single file called `console` somewhere under the root dir - which is a convention that not all projects might follow.